samples/bluetooth/peripheral_hids_mouse/README.rst

```diff
@@ -1,24 +1,134 @@
-.. _peripheral_hids:
+.. _peripheral_hids_mouse:
 
-Bluetooth: Peripheral HIDs mouse
+Bluetooth: Peripheral HIDS mouse
 ################################
 
+The Peripheral HIDS mouse sample demonstrates how to use the :ref:`hids_readme` to implement a mouse input device that you can connect to your computer.
+
 Overview
 ********
 
-Similar to the :ref:`Peripheral <ble_peripheral>` sample, except that this
-application specifically exposes the HID GATT Service. The report map used is
-for a generic mouse.
+The sample uses the buttons on a development board to simulate the movement of a mouse.
+The four buttons simulate movement to the left, upward, to the right, and downward, respectively.
+Mouse clicks are not simulated.
+
+This sample is similar to Zephyr's :ref:`zephyr:ble_peripheral` sample, except that it specifically exposes the HID GATT Service.
```

Comment:
I'm not sure - this sentence is from the original input.

Looks like it's copied from the Zephyr docs: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/samples/bluetooth/peripheral_hids/README.html 

I think this info can be helpful if the sample actually IS similar - but it has probably diverged a log.

@kapi-no You added this sentence initially - what do you think?

Reply:
Yes, it was copied from Zephyr. 

Well, every BLE sample is somewhat similar to the Zephyr one, so I guess this part of the sentence is not needed. 
And It will be aligned with other NCS sample (e.g. we don't have this sentence in the peripheral_nus)

Reply:
OK, changed to "This sample exposes the HID GATT Service."